FFXIVClientStructs/FFXIV/Client/UI/Misc/AcquaintanceModule.cs

```diff
@@ -27,6 +27,9 @@ public struct Acquaintance {
         [FieldOffset(0xD8)] public ulong ContentId;
     }
 
+    [MemberFunction("44 8B 89 ?? ?? ?? ?? 4C 8B C1 45 85 C9")]
+    public partial Aquaitence* GetTellHistory(int index);
```

Comment:
@wolfcomp can't type and I copied it wrong. gg no re
```suggestion
    public partial Acquaintance* GetTellHistory(int index);
```

Reply:
Whops, but you should know by now to not trust what I type as a copy-paste <.<